<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mp3播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/0.ico</normaloff>:/images/0.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>播放列表</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>611</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="open_btn">
       <property name="text">
        <string>打开</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/1.GIF</normaloff>:/images/1.GIF</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play_btn">
       <property name="text">
        <string>播放</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/4.bmp</normaloff>:/images/4.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause_btn">
       <property name="text">
        <string>暂停</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/5.bmp</normaloff>:/images/5.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_btn">
       <property name="text">
        <string>停止</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/6.bmp</normaloff>:/images/6.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prev_btn">
       <property name="text">
        <string>上一曲</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/7.bmp</normaloff>:/images/7.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_btn">
       <property name="text">
        <string>下一曲</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/8.bmp</normaloff>:/images/8.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sound_btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/volumn.bmp</normaloff>:/images/volumn.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sound_slider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>歌曲名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="time_slider">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time_label">
       <property name="text">
        <string>播放进度</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
